config/locales/es/activerecord.yml

```diff
@@ -151,7 +149,7 @@ es:
         description: "Descripción"
         external_url: "Enlace a documentación adicional"
         administrator_id: "Administrador"
-        location: "Información adicional de la ubicación"
+        location: "Ubicación (opcional)"
```

Comment:
Since we manage Spanish translations through code, IMHO we shouldn't handle them through crowdin :thinking:.

I'd remove all changes in `config/locales/es/` from this pull request. If some of them make sense, then we could open a different pull request for them.

What do you think?

Reply:
Yes, at first i thought the same, but then i realized i will always have to remember to remove any changes to Spanish translations that Crowdin introduces and this and future PR's, so i thought maybe i can keep Spanish translations updated from the Crowdin side and let Crowdin to do the commits. My intention was to save myself time, but not sure if this could lead us to errors like this one you detected.

Reply:
Finally removed all changes to Spanish locale so this is not a problem anymore!